_scripts/templates/Dart/cli.dart

```diff
@@ -9,8 +9,8 @@ import 'dart:convert';
 <if (case_insensitive_type)>
 
 class CaseChangingCharStream extends CharStream {
-        CharStream stream;
-        bool upper;
+    CharStream stream = InputStream.fromString('');
+    bool upper = false;
```

Comment:
Such dummy initialization looks weird.

I think it makes sense to copy this class from https://github.com/antlr/antlr4/blob/master/doc/resources/case_changing_char_stream.dart

Reply:
I did copy the code, but I see, they changed the code in the last two months.

When the build settles down, I'll make the change.